graphql_compiler/query_formatting/common.py

```diff
@@ -28,6 +28,63 @@ def _raise_invalid_type_error(name, expected_python_type_name, value):
                                                          type(value).__name__))
 
 
+def _deserialize_anonymous_json_argument(expected_type, value):
+    """Deserialize argument. See docstring of deserialize_json_argument."""
+    allowed_types_for_graphql_type = {
+        GraphQLDate: (six.string_types,),
+        GraphQLDateTime: (six.string_types,),
+        GraphQLFloat: (six.string_types, float, six.integer_types),
+        GraphQLDecimal: (six.string_types, float, six.integer_types),
+        GraphQLInt: (six.integer_types,),
```

Comment:
Should we allow int-like strings here as well? I believe JSON as a format doesn't actually support the full int64 range, since Javascript i.e. JSON only knows about doubles, and any int above `2**53` would lose precision as a result. The easy-ish fix here is to allow strings that parse into ints to work.